### ISSUE TITLE
Curl links to various system libraries

### ISSUE THREAD

**Issue by vbraun** [hashdist/hashstack#880]

**Comment by ahmadia** [comment]
> Thats potentially a problem for reproducible builds if it drags in that many host specifics?

Absolutely, thanks for noting this.  I'd say maybe half of those are fairly portable libraries, but I don't know/recognize the other half.

**Comment by jcftang** [comment]
I've tried packaging nss and nspr but I wasn't too sucessful the last time I tried

**Comment by certik** [comment]
That's definitely a problem --- the philosophy of Hashdist is that if you want to link to system wide libraries, you can. But if you don't want to, you can build everything from source. So we have to fix this.
